pkg/apis/k0s/v1beta1/api.go

```diff
@@ -57,11 +60,12 @@ func DefaultAPISpec() *APISpec {
 	addresses, _ := iface.AllAddresses()
 	publicAddress, _ := iface.FirstPublicAddress()
 	return &APISpec{
-		Port:       defaultKasPort,
-		K0sAPIPort: 9443,
-		SANs:       addresses,
-		Address:    publicAddress,
-		ExtraArgs:  make(map[string]string),
+		Port:        defaultKasPort,
+		K0sAPIPort:  9443,
+		BindAddress: "0.0.0.0",
```

Comment:
Any reason for defaulting this to "0.0.0.0"?

Reply:
Compared to just leaving it out?

Reply:
Yes.